Sources/Quick/Configuration/Configuration.swift

```diff
@@ -72,10 +72,16 @@ final public class Configuration: NSObject {
         provided with metadata on the example that the closure is being run
         prior to.
     */
+#if !os(Linux)
```

Comment:
I see quite a few conditional macros in the Swift codebase along the lines of "if the Objective-C runtime is available". Could we change this to something like that? The rationale being that what we really want to check here is whether `@objc` is available, not what the operating system is. Even now this isn't completely correct--what if the OS is FreeBSD? :wink: